envoy/assets/dashboards/overview_v2.json

```diff
@@ -0,0 +1,1320 @@
+                    }
+                  ],
+                  "style": {
+                    "palette": "dog_classic",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "line"
+                }
+              ],
+              "yaxis": {
+                "include_zero": true,
+                "scale": "linear",
+                "min": "auto",
+                "max": "auto"
+              },
+              "markers": []
+            },
+            "layout": {
+              "x": 0,
+              "y": 9,
+              "width": 8,
+              "height": 3
+            }
+          },
+          {
+            "id": 8582025331671068,
+            "definition": {
+              "title": "Cluster Load Balancer Panics",
+              "title_size": "16",
+              "title_align": "left",
+              "show_legend": true,
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "alias": "total_requests",
+                      "formula": "query1"
+                    }
+                  ],
+                  "response_format": "timeseries",
+                  "queries": [
+                    {
+                      "query": "avg:envoy.cluster.lb_healthy_panic{*,datacenter:us1.prod.dog,*,$release} by {release}.as_count()",
+                      "data_source": "metrics",
+                      "name": "query1"
+                    }
+                  ],
+                  "style": {
+                    "palette": "warm",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "line"
+                }
+              ],
+              "yaxis": {
+                "include_zero": true,
+                "scale": "linear",
+                "label": "",
+                "min": "auto",
+                "max": "auto"
+              },
+              "markers": []
+            },
+            "layout": {
+              "x": 0,
+              "y": 12,
+              "width": 5,
+              "height": 3
+            }
+          },
+          {
+            "id": 6140256689199842,
+            "definition": {
+              "type": "note",
+              "content": "**Cluster load balancer (lb) panics** refer to how many times the proxy is ignoring the health check restrictions because there are not enough healthy upstreams. \n\nIt's helpful to see when Envoy is redirecting to unhealthy upstreams which should not happen normally. It's also useful to see when health checks are not properly set up. Cluster panics should always be 0.",
```

Comment:
```suggestion
              "content": "**Cluster load balancer (lb) panics** refer to how many times the proxy is ignoring the health check restrictions because there are not enough healthy upstreams. \n\nIt's helpful to see when Envoy is redirecting to unhealthy upstreams, which should not happen normally. It's also useful to see when health checks are not properly set up. Cluster panics should always be 0.",
```